bin.src/generateCoefficients.py

```diff
@@ -87,25 +87,33 @@
     else:
         tStart = args.tStart
 
+    # We have already ruled out the case where args.tSpan and args.tEnd are None, above.
```

Comment:
I would like to -- but we can't. We don't know tStart (unless the user provides it) until after we read the orbits file.
But testing that both tEnd and tStart are not None can usefully (as you pointed out) be done before we read the orbits file. 
So they end up separated by necessity (unless you require that the user specify tStart always).